CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Added validation to ensure no roles or bindings have cluster-admin permissions.
```

Comment:
I think I need some more context in this PR :slightly_smiling_face: 
Which resources are being reconciled/checked here that we want to validate?
Is it all the roles and bindings that rbac-operator is creating or is it all the ones that exist on the cluster? (since we need to set some admin permissions too, right?)